### ISSUE TITLE
Depricated variables in mysql 8

### ISSUE THREAD

**Issue by waynegemmell** [saltstack-formulas/mysql-formula#188]
/var/local/salt/formulas/mysql-formula/mysql/defaults.yaml:        query_cache_limit: 1M
/var/local/salt/formulas/mysql-formula/mysql/defaults.yaml:        query_cache_size: 16M

**Comment by noelmcloughlin** [comment]
yes, both are deprecated: https://dev.mysql.com/doc/mysqld-version-reference/en/mysqld-version-reference-optvar-changes-8-0.html

**Comment by waynegemmell** [comment]
Agreed, they are set in the formula though and need to be removed.

**Comment by noelmcloughlin** [comment]
I'm willing to fix - just waiting for #186 to get approved/merged first.
